redirects.csv

```diff
@@ -47,7 +47,7 @@ https://discipleship.crossroads.net/*,https://www.crossroads.net/huddle,301!,mas
 /childcare,https://${env:CRDS_ANGULAR_ENDPOINT}/childcare,200!
 /childcare/*,https://${env:CRDS_ANGULAR_ENDPOINT}/childcare/:splat,200!
 /childcare2/*,https://${env:CRDS_CHILDCARE_ENDPOINT}/:splat,200!
-/corkboard*,https://${env:CRDS_ANGULAR_ENDPOINT}/corkboard:splat,200!
```

Comment:
Not sure if this is ready for production, working with @tatelucas to see if it is.